src/modules/postScores.js

```diff
@@ -0,0 +1,26 @@
+const postData = async (user, score) => {
+  try {
+    const userData = {
+      name: 'My leatherboard',
+      return: 'Game with ID: CobbyElson added.',
+      user,
+      score,
+    };
+
+    const data = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/CobbyElson/scores', {
+      method: 'post',
+      body: JSON.stringify(userData),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+    if (data.status === 201) {
+      const successMess = document.querySelector('.successMess');
+      successMess.textContent = 'Score added successfully';
+    }
```

Comment:
- [ ] Excellent work on displaying the success message when the score is created. Your implementation shows promise! However, I believe there's room for improvement by incorporating a `setTimeout()` function.

   - Using the `setTimeout()` function will enhance the user experience by controlling how long the message remains visible on the UI. You can specify a desired time interval for the message to appear and then automatically disappear after the set duration.
   - By incorporating `setTimeout()`, you can tailor the message display time as needed and ensure that it does not clutter the UI.